File: API/AuthService/Models/SaveTokenRequest.cs
using System.ComponentModel.DataAnnotations;

namespace AuthService.Models
{
    public class SaveTokenRequest
    {
        [Required]
        public string Username { get; set; }

        [Required]
        public string Role { get; set; } // Client, Waiter, Courier, Cook

        [Required]
        public string Token { get; set; }
    }
}

File: ClientSide/UkrainianStyleRestaurant_2.0_another/app/src/main/java/com/example/ukrainianstylerestaurant/model/SaveTokenRequest.java
package com.example.ukrainianstylerestaurant.model;

public class SaveTokenRequest {
    public String username;
    public String role;
    public String token;

    public SaveTokenRequest(String username, String role, String token) {
        this.username = username;
        this.role = role;
        this.token = token;
    }
}